### ISSUE TITLE
Ozone instance not responding to label changes, moderation queue emptied after container restart

### ISSUE THREAD

**Issue by Nsbx** [bluesky-social/ozone#228]
Since yesterday evening, my Ozone instance has stopped "working" correctly. Despite adding or removing labels, no changes appear to take effect.

In an attempt to resolve the issue, I restarted the container. However, after rebooting, I found that my moderation queue had been completely cleared. Is this expected behavior?

Aside from the empty moderation queue, could the initial issue with unresponsive labels be due to a sudden influx of new users?

Steps to reproduce:

1. Attempt to add or remove labels on the instance.
2. Observe that no changes take effect.
3. Restart the container.
4. Check the moderation queue (noting that it is empty after restart).

Expected behavior:

- Label changes should apply as expected.
- The moderation queue should persist after a container restart.

Thank you in advance for your assistance.

**Comment by Michal-Tomasz** [comment]
Hi, did you manage to solve the problem with the labels? I add labels in ozone but no changes happen in the application. I have no idea anymore, maybe you can guide me?

**Comment by SteveTownsend** [comment]
I hit this initially because my labels were malformed (mixed case).
Once I got the config JSON set up right, it worked fine

**Comment by Michal-Tomasz** [comment]
and you can send me a fragment of your JSON or I can send you mine because it seems to be OK and yet it does not work properly

**Comment by Michal-Tomasz** [comment]
yes the labels are visible in the account, the default values and everything is visible. So I have to look somewhere else. I found such a problem maybe it is here?
@repo undefined

https://imgur.com/QPR3ADd

**Comment by SteveTownsend** [comment]
That looks suspect for sure. I have not seen it myself.
